app/Policies/WebhookPolicy.php

```diff
@@ -11,30 +11,29 @@ class WebhookPolicy
 {
     use HandlesAuthorization;
 
-    /**
-     * Determine whether the user can update the webhook.
-     *
-     * @param  \App\Models\User  $user
-     * @param  \App\Webhook  $webhook
-     * @return mixed
-     */
-    public function update(User $user, Webhook $webhook)
-    {
-        return $user->id === $webhook->user_id;
-    }
+    /**	
+     * Determine whether the user can update the webhook.	
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.NonIndentTabsUsed</br>**Severity**: error</br>**File**: app/Policies/WebhookPolicy.php</br>**Line**: 15</br>Spaces must be used for alignment; tabs are not allowed